Added help section: Options

wernert opened the pull request:
WIP , please review

wernert committed:
Added Section: Options

WIP , please review

jonschlinkert commented on .verb.md:
```diff
@@ -66,6 +66,26 @@ $ copy *.js foo
+    'dist/client',
+    {"ignore":[ "src/client/scripts/**/*.*","src/client/styles/**/*.*"]},
+    }
+  );
```
Please format all on one line, thanks!

wernert committed:
Update .verb.md

sorry , i had removed the callback function for the snippet 
and now just realized: 4th Argument, callback function, is required with this notation, right?
So i have made 3 versions in the commit, different verbosity level, please choose.
